matching-tools/pom.xml

```diff
@@ -0,0 +1,46 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<name>GraphHopper Map Matching Tools</name>
+	<groupId>com.graphhopper</groupId>
+	<artifactId>graphhopper-map-matching-tools</artifactId>
```

Comment:
I know you wanted to make it identical to the GH core modules. But is a separate module necessary just for one class?

My intend for the separate tools module in GH was to separate swing usages from core and other deps